README.md

```diff
@@ -78,4 +78,50 @@ Not supported
 | storage-driver-buffer-memory | long | 33554432 | The total bytes of memory the producer can use to buffer records waiting to be sent to the server. |
 | storage-driver-compression-type | string | none | The compression type for all data generated by the producer. The default is none (i.e. no compression). Valid values are none, gzip, snappy, lz4, or zstd.  |
 | storage-net-node-addrs | list | "" | A list of host/port pairs to use for establishing the initial connection to the Kafka cluster.  This list should be in the form *host1:port1*,*host2:port2* |
-| storage-net-node-port | integer | 9092 | The common port number to access the storage nodes, may be overriden adding the port number to the storage-driver-addrs, for example: "127.0.0.1:9020,127.0.0.1:9022,..." |
\ No newline at end of file
+| storage-net-node-port | integer | 9092 | The common port number to access the storage nodes, may be overriden adding the port number to the storage-driver-addrs, for example: "127.0.0.1:9020,127.0.0.1:9022,..." |
+
+## Custom Kafka Headers
+
+Scenario example:
+
+```javascript
+var customKafkaHeadersConfig = {
+    "storage" : {
+        "net" : {
```

Comment:
let's use `storage-driver-request-headers` either `storage-driver-create-headers` option
also, the doc is updated, but the option is not added into the defaults/schema yaml files
moreover, the declared configuration option should be handled by the driver's code